cpu/cpu_test.go

```diff
@@ -160,7 +158,7 @@ func (cis *CPUInfoSuite) TestGetMetricTypes() {
 
 				namespaces := []string{}
 				for _, m := range mts {
-					namespaces = append(namespaces, strings.Join(m.Namespace().Strings(), "/"))
+					namespaces = append(namespaces, strings.Join(m.Namespace.Strings(), "/"))
```

Comment:
Keeping this intact as it's causing tests to fail with the following error: 
Expected the container ([]string) to contain: 'intel/procfs/cpu/*/user_percentage' (but it didn't)!